src/main/telemetry/telemetry.h

```diff
@@ -59,5 +59,4 @@ bool telemetryDetermineEnabledState(portSharing_e portSharing);
 
 void telemetryUseConfig(telemetryConfig_t *telemetryConfig);
 
-#define TELEMETRY_SHAREABLE_PORT_FUNCTIONS_MASK (FUNCTION_TELEMETRY_FRSKY | FUNCTION_TELEMETRY_LTM)
-
+#define TELEMETRY_SHAREABLE_PORT_FUNCTIONS_MASK (FUNCTION_TELEMETRY_FRSKY | FUNCTION_TELEMETRY_LTM | FUNCTION_TELEMETRY_CRSF)
```

Comment:
> @mikeller , I must admit I don't fully understand the sharable aspects of telemetry. What criteria does a telemetry implementation have to satisfy to be shareable?

On the technical side the only criteria are that the protocol has to only require the Tx pin of the FC's UART, so that the serialRx protocol can have exclusive use of the Rx pin, and that it has to be capable of using the UART at whatever speed the serialRx protocol dictates.

From what you are saying CRSF telemetry seems to meet both of those.

It could also be argued that for tightly integrated solutions like CRSF, where users wouldn't be expected to run CRSF telemetry with a serialRx that is not CRSF, it would be more userfriendly to go the way that JETIEXBUS goes and don't even show the telemetry protocol in the UI, but activate it automatically whenever the serialRx protocol is selected.

I don't know enough about CRSF to know which of the two cases above is best.

> So what is special about FrSky and LTM telemetry that allow them to be shared with serial RX? For example, why cant Mavlink telemetry be shared?

FrSky and LTM were the only two protocols implemented at the time when I added sharing that used only the Tx pin. If I remember correctly, Mavlink wasn't there back then, and if it only uses Tx it should probably be added to the list.